canonical-web-and-design/ubuntu.com
# Add /openstack/support page

## Done

- Added new page at /openstack/support, based on the [copy doc](https://docs.google.com/document/d/1mHaZQ_VZnISTUEpYKiVK-Ejr5ne8lVevpxEc0LpXYKI/edit#)
- Removed an old redirect that sent users from /openstack/support to /advantage
- Added "Support" to the mega nav and the footer navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11166.demos.haus/openstack/support
    - Be sure to test on mobile, tablet and desktop screen sizes
- Make sure that the page matches the copy doc and the links work
- Check that the meganav includes links to the new page

## Issue / Card

Fixes [#4686](https://github.com/canonical-web-and-design/web-squad/issues/4686)

## Screenshots
![screencapture-0-0-0-0-8001-openstack-support-2022-01-19-12_17_06 (1)](https://user-images.githubusercontent.com/2376968/150128060-fd24a7d2-31d4-4f75-a424-c9aab269c206.png)
